templates/borg-backup.sh.epp

```diff
@@ -180,7 +180,7 @@ backup_borg() {
   # Change the working directory in case the source paths are relative
   cd "<%= $working_directory %>"
 
-  borg create "${options[@]}" --stats --verbose "$dst::backup-$(date "+%Y%m%d-%H%M%S")" "${src[@]}"
+  borg create "${options[@]}" --stats --verbose "$dst::backup-$(date "+%Y%m%d-%H%M%S")" "${src[@]}" || true
```

Comment:
That will always cause successful exit. Don't you only want successful exit when pruning is enabled?

Reply:
yeah.. if it's important to fail in other cases it might be better to add `|| true` conditionally.. e.g.
```
borg create ... "${src[@]}"<% if manage_prune { %>|| true<% } %>
```

Reply:
should be fine now